.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -158,6 +158,17 @@ private object OdysseyStats : WPComPluginBuild(
 	pluginSlug = "odyssey-stats",
 	archiveDir = "./dist/",
 	docsLink = "PejTkB-3N-p2",
+	buildSteps = {
```

Comment:
I assumed the step would be run after `yarn build`. Please do correct me if I'm wrong thanks friends 😄

Reply:
Ah it looks like so:

https://github.com/Automattic/wp-calypso/blob/b44817419dc5b84ec704c86c5180d3c423cbf860/.teamcity/_self/lib/wpcom/WPComPluginBuild.kt#L116